database.sql

```diff
@@ -0,0 +1,42 @@
+-- sqlite3 database.sqlite < database.sql
+
+CREATE TABLE USERS (
+  user_id INTEGER PRIMARY KEY,
+  first_name TEXT NOT NULL,
+  last_name TEXT NOT NULL,
+  email TEXT NOT NULL UNIQUE,
+  username TEXT NOT NULL UNIQUE,
+  password TEXT NOT NULL,
+  usertype_id INTEGER,
+  FOREIGN KEY (usertype_id) REFERENCES USERTYPE (usertype_id)
+);
+
+CREATE TABLE SESSIONS (
+  session_id INTEGER PRIMARY KEY,
```

Comment:
I think that this instruction probably begin the counter in 0 instead of 1 like is specified in database.go lines 119 and 120 in addUser (last argument)

Reply:
Good catch. Fixed

Reply:
I don't think so. But, does it matter?

Reply:
I don't know, I thought that the ID in tables begins in 0, like array's index